tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -19,17 +19,17 @@
 nigtly_runtime_heat_duration = os.environ.get('NIGHTLY_RUNTIME_HEAT_DURATION', '5')
 
 tf_hub_cache_dir = os.environ.get('TFHUB_CACHE_DIR',
-                                  os.path.join(tempfile.gettempdir(), "tfhub_modules"))
-hf_cache_dir = os.environ.get('HF_HUB_CACHE',
-                              os.path.join(tempfile.gettempdir(), "hugging_face"))
+                                  os.path.join(tempfile.gettempdir(), "tfhub_models"))
+tf_hfhub_cache_dir = os.environ.get('TF_HFHUB_CACHE_DIR',
+                                    os.path.join(tempfile.gettempdir(), "tf_hfhub_models"))
 os.environ['TFHUB_CACHE_DIR'] = tf_hub_cache_dir
-os.environ['HF_HUB_CACHE'] = hf_cache_dir
+os.environ['TF_HFHUB_CACHE_DIR'] = tf_hfhub_cache_dir
 
 no_clean_cache_dir = False
-hf_hub_cache_dir = tempfile.gettempdir()
+pt_hfhub_cache_dir = os.path.join(tempfile.gettempdir(), "pf_hfhub_models")
 if os.environ.get('USE_SYSTEM_CACHE', 'True') == 'False':
     no_clean_cache_dir = True
-    os.environ['HUGGINGFACE_HUB_CACHE'] = hf_hub_cache_dir
+    os.environ['PT_HFHUB_CACHE_DIR'] = pt_hfhub_cache_dir
```

Comment:
The original plan was to split cached TF and PR models into two separate folders but it looks impossible because tf_sentence_transformers package relies on HF env variable and does not have like `cache_dir` option for loading models.
So this PR is converted to just simple fix.